contrib/install.sh

```diff
@@ -94,11 +94,11 @@ then
   pkg=deb
   distribution=ubuntu
   version=$(grep VERSION_CODENAME /etc/os-release | cut -d= -f2)
-  if [ "$version" != focal ] && [ "$version" != bionic ] && [ "$version" != xenial ];
+  if [ "$version" != jammy ] && [ "$version" != focal ] && [ "$version" != bionic ] && [ "$version" != xenial ];
```

Comment:
The latest refresh of [Xenial went EOL about 18 months ago](https://endoflife.software/operating-systems/linux/ubuntu). No need to do anything now, but we should consider dropping it.